### DISCUSSION TITLE
Unable to take a photo with my back camera or upload a photo from my photos library

### DISCUSSION THREAD

**Discussion by UniqueApricot8662** [community/community#75362] [GitHub Education]
Hello,

I am trying to apply for GitHub education benefits, such as GitHub pro and copilot.

I am on iOS at the moment and don't have the ability to take a photo with my back camera. This poses several issues such as positioning my screen correctly, quality etc.

I also can't add my school mail account because it won't me receive emails from people who aren't in my 'organization'. Will I still be able to receive GitHub educational benefits?

Can I send the screenshot via email to somebody? Or is there a way to use the back camera on the website?

![image](https://github.com/community/community/assets/145867345/010be714-db6a-4974-a85b-bfe8ead9563a) (screenshot of what I see)

Thanks

**Comment by seldomgreeen** [comment]
Same issue with me. It seems that it's an issue on github's side. I'd recommend sending them an email.

**Comment by huongtr** [comment]
same issue, it's very bad UX, trying to take a clear picture but no hope

**Comment by h1ac** [comment]
I used the opera browser and It let me choose the back camera! I got accepted :)
